engines/bastion_katello/app/assets/javascripts/bastion_katello/subscriptions/manifest/views/manifest-import-history.html

```diff
@@ -8,7 +8,7 @@
   <tbody>
     <tr ng-repeat="status in statuses track by $index">
         <td>
-          <i ng-class="status.status == 'FAILURE' ? 'fa fa-warning' : fa fa--ok'" />
+          <i ng-class="status.status == 'FAILURE' ? 'fa fa-warning' : 'fa fa-check'" />
```

Comment:
Broken since Oct 2014, this typo caused js error and was not showing the small icons in the events list.